GITHUB101.md

```diff
@@ -0,0 +1,94 @@
+This document is aimed at getting started with github if you never used it.
+
+If you're a developer there's more info at [README.md](https://github.com/Western-Health-Covid19-Collaboration/wh_covid19_app/blob/master/README.md).
+
+# Quick links
+
+- [Issues assigned to Gregg](https://github.com/Western-Health-Covid19-Collaboration/wh_covid19_app//issues?q=assignee%3Agreggmiller+is%3Aopen)
+
+- [Issues tagged with design](https://github.com/Western-Health-Covid19-Collaboration/wh_covid19_app//labels/design) need designer's attention
```

Comment:
Can you also add a link for issues tagged with `Ready for dev` ?

Reply:
I added it but devs shouldn't be reading this doc. In any case I'm also adding a link to tag "good first issue".

Reply:
Oh, thats fantastic 🙌